gateway-server/src/main/java/org/apache/knox/gateway/websockets/GatewayWebsocketHandler.java

```diff
@@ -173,12 +174,41 @@ private synchronized String getMatchedBackendURL(final String path) {
 
     /* URL used to connect to websocket backend */
     String backendURL = urlFromServiceDefinition(serviceRegistryService, entry, path);
+    LOG.logMessage("Url obtained from services definition: " + backendURL);
```

Comment:
Use a new function to log generic log messages, `LOG.logMessage` is used to log WS messages.

Reply:
Done.